### PR TITLE
Dustin's paper feedback

### PR DESCRIPTION
**Pull request by dabreegster**

### CODE REVIEW

File: vignettes/paper.Rmd [hunk 1]
```diff
@@ -174,7 +174,8 @@ Finally, in Section 4, we discuss limitations of the approach and possible direc
 
 The aim of the ClockBoard zoning system is to tackle the issues associated with available zoning systems and to provide a standard template for analysis of the geographic distribution of urban phenomena between cities internationally.
 As outlined above, there is a need for open access zoning systems that are readily available and consistent.
-The requirements of urban analysts, geographers, transport modellers and other working with geographic data across cities are diverse, however, motivating our approach which is free, open, reproducible and easy to extend: modifications to the ClockBoard system and completely new zoning systems could build the approach.
```
Comment (dabreegster) [inline vignettes/paper.Rmd:177]: See comments below, I'm not sure at all what you're trying to say here

File: vignettes/paper.Rmd [hunk 2]
```diff
@@ -36,7 +36,7 @@ abstract: |
   often based on somewhat arbitrary factors; 2) evidence-based zoning
   systems are often highly variable in size and shape, reducing their
   utility for inter-city comparison; and 3) official zoning systems are non-existent,
-  not publicly available, or are too course, hindering urban analysis
+  not publicly available, or are too coarse, hindering urban analysis
```
Comment (Robinlovelace) [inline vignettes/paper.Rmd:39]: :+1:

File: vignettes/paper.Rmd [hunk 3]
```diff
@@ -154,7 +154,7 @@ The motivations for generating a new zoning system and use cases envisioned incl
 -   Locating cities.
     Automated zoning systems based on a clear centrepoint can support map interpretation by making it immediately clear where the city centre is, and what the scale of the city is.
 
--   Reference system of everyday live.
+-   Reference system for everyday life.
```
Comment (Robinlovelace) [inline vignettes/paper.Rmd:157]: :+1: woops sloppy typos!

File: vignettes/paper.Rmd (hunk 1 above)
Comment (Robinlovelace) [reply vignettes/paper.Rmd:177]: Yes it's not a very clear couple of sentences, good point. Essentially yes, our aim is for ClockBoard to be useful for many people. Over the last few days I've thought it useful to refer to at least 3 levels of abstraction: zoning systems being the highest level, segmented annuli being a type of zoning system, and ClockBoard being an implementation of the segmented annuli with specific parameters. Possibly not worth saying that here but worth being clear and I think the essence of what I'm trying to say can be communicated with fewer words.

File: vignettes/paper.Rmd [hunk 4]
```diff
@@ -282,7 +283,7 @@ install.packages("zonebuilder")
 To make the project and resulting zones available to more people, we subsequently developed a Rust crate, that enables the creation of ClockBoard zones on all major platforms using a command line interface.
 See [github.com/zonebuilders/zonebuilder-rust](https://github.com/zonebuilders/zonebuilder-rust) for details.
 
-To enable creation of ClockBoard zones for non-programmers and to encourage people to explore alternative zoning systems, we created a simple web application hosted at [zonebuilders.github.io/zonebuilder-rust/](https://zonebuilders.github.io/zonebuilder-rust/) (see Figure \@ref(fig:interactive)).
+To enable creation of ClockBoard zones for non-programmers and to encourage people to explore alternative zoning systems, we created a simple web application hosted at [zonebuilders.github.io/zonebuilder-rust/](https://zonebuilders.github.io/zonebuilder-rust/) (see Figure \@ref(fig:interactive)). This app leverages the Rust implementation for generating ClockBoards, compiling it to WebAssembly and using the standard open-source [leafletjs.com](https://leafletjs.com) mapping library as an interface.
```
Comment (Robinlovelace) [inline vignettes/paper.Rmd:286]: Thanks, this is great!

**Review by Robinlovelace — APPROVED** [review]

**Comment by Robinlovelace** [comment]
Thanks for the comments @dabreegster, looking good. Do you want a bash at updating the garbled sentence? Happy to give it a bash this side also but I think what you put in the comment gets close to what we mean...

**Comment by Robinlovelace** [comment]
Just merged. Thanks Dustin!

**Pull request merged**